dissect/target/plugins/general/plugins.py

```diff
@@ -94,15 +101,128 @@ def get_description_dict(
     return output_descriptions
 
 
+@dataclass
+class PluginFunction:
+    name: str
+    output_type: str
+    class_object: str
+    method_name: str
+    plugin_desc: dict
+
+
 class PluginListPlugin(Plugin):
     def check_compatible(self):
         return True
 
-    @export(output="none")
-    @arg("--docs", dest="print_docs", action="store_true")
-    def plugins(self, print_docs=False):
-
-        categorized_plugins = dict(sorted(categorize_plugins().items()))
+    @export(output="none", cache=False)
+    def plugin_function_db(self) -> Tuple[dict, list]:
+        db = []
+        rootset = set()
+
+        def all_plugins():
```

Comment:
Does `plugin.plugins()` not give the same result?

Reply:
Not (all) osplugins and childplugins